### PR TITLE
fixes #3709 - explicitly say puppet environments vs just 'environments'

### PR DESCRIPTION
**Pull request by ohadlevy**

### CODE REVIEW

File: app/views/environments/index.html.erb [hunk 1]
```diff
@@ -1,27 +1,27 @@
...
+        <%= action_buttons(link_to(_('Classes'), puppetclasses_path(:search => "environment = #{environment}")),
+                           display_delete_if_authorized(hash_for_environment_path(:id => environment.name), :confirm => "Delete #{environment.name}?")) %>
       </td>
     </tr>
   <% end %>
 </table>
 
 <%= page_entries_info @environments %>
-<%= will_paginate @environmentsS %>
+<%= will_paginate @environments %>
```
Comment (lzap) [inline app/views/environments/index.html.erb:27]: Huh this looks like typo in develop?

Testing...

File: app/views/environments/index.html.erb [hunk 2]
```diff
@@ -1,27 +1,27 @@
-<% title _("Environments") %>
+<% title _('Puppet Environments') %>
 
-<% title_actions import_proxy_select(hash_for_import_environments_environments_path), button_group(display_link_if_authorized(_("New Environment"), hash_for_new_environment_path)) %>
+<%= environments_title_actions %>
 
 <table class="table table-bordered table-striped table-two-pane">
   <tr>
-    <th><%= sort :name, :as => s_("Environment|Name") %></th>
+    <th><%= sort :name, :as => s_('Environment|Name') %></th>
```
Comment (lzap) [inline app/views/environments/index.html.erb:7]: I keep doing this all the time, but I think - it makes no difference :-)

**Comment by lzap** [comment]
In the menu there is still Environments (under Puppet section tho). Intentional?

Tested, works fine. Good for merge.

File: app/views/environments/index.html.erb (hunk 1 above)
Comment (ohadlevy) [inline app/views/environments/index.html.erb:27]: it seems like it was there since 2010. commited at 5f6b2196

**Comment by ohadlevy** [comment]
@lzap, the menu remained the same, as its under the puppet category in the dropdown.

**Comment by domcleal** [comment]
Thanks @ohadlevy, merged as 72479d52f5 for 1.4.

**Pull request closed without merging**